### ISSUE TITLE
Google Play requires that apps target API level 29 or higher.

### ISSUE THREAD

**Issue by awebi** [mixpanel/mixpanel-react-native#130]
I'm trying to create an android build, but I'm getting the error below:

```
* What went wrong:
Execution failed for task ':mixpanel-react-native:lint'.
> Lint found errors in the project; aborting build.
  
  Fix the issues identified by lint, or add the following to your build script to proceed with errors:
  ...
  android {
      lintOptions {
          abortOnError false
      }
  }
  ...
  Errors found:
  
  <project-directory>/node_modules/mixpanel-react-native/android/build.gradle:18: Error: Google Play requires that apps target API level 29 or higher.
   [ExpiredTargetSdkVersion]
          targetSdkVersion 26
          ~~~~~~~~~~~~~~~~~~~
```

Library Versions:

"mixpanel-react-native": "^1.3.9",
"react": "17.0.1",
"react-native": "0.64.1",

**Comment by awebi** [comment]
Hi @zihejia, any help on this will be really appreciated.

**Comment by zihejia** [comment]
hi @awebi , we will upgrade this. Thanks for bringing it up!

**Comment by awebi** [comment]
> hi @awebi , we will upgrade this. Thanks for bringing it up!

Thanks @zihejia. Looking forward to the update. Cheers!

**Comment by zihejia** [comment]
hi @awebi , we have released 1.3.10 which has the fix, please feel free to reopen if the problem still exists.
